src/tests/request.test.js

```diff
@@ -262,6 +293,40 @@ describe('/api/v1/requests', () => {
       res.should.have.status(404);
       expect(JSON.parse(res.text).error).to.equal('You\'ve not made any requests');
     });
+
+    it('should retrieve all open requests made by manager\'s direct report', async () => {
+      const res = await chai.request(app)
+        .get(`${URL_PREFIX}/pending`)
+        .set('token', manager.body.data.token);
+
+      res.should.have.status(200);
```

Comment:
Check `res.body.data` length of array to ensure that it is returning the actually amount of data

Reply:
OK, I will do that